_technical/cicd-as-platform.md

```diff
@@ -4,52 +4,43 @@ description: The CI/CD system is treated as a platform on top of the cloud platf
 layout: pattern
 ---
 
-![CI/CD as Platform]({{ site.baseurl }}/assets/images/Generic.png)
-
-The complexity of software delivery in an enterprise company can easily reach a level when
-a dedicated team would have to be responsible for its maintenance.
-
-## Context
-
-Enterprise company is developing Cloud Native CI/CD services during a transormation or
+An Enterprise company is developing Cloud Native CI/CD services during a transformation or
 cloud migration. CI/CD services include building of container images, code quality checks,
 security scans and deployment of artifacts to environments. CI/CD plays a significant
 part in compliance requirements and making sure production systems are secure.
 
-## Problem
+![CI/CD as Platform]({{ site.baseurl }}/assets/images/cicd.png)
```

Comment:
The image should be the first text after the meta-data section at the top to follow the standard pattern. This applies to all of the patterns.

Reply:
![Uploading Screenshot 2020-12-14 at 15.25.59.png…]()

Reply:
![Screenshot 2020-12-14 at 15 25 59](https://user-images.githubusercontent.com/57802771/102101797-fd097e00-3e22-11eb-9a3d-e3fa8a42185b.png)